docs/release/release-process.md

```diff
@@ -192,6 +201,77 @@ success, all release artifacts will be officially published.
 After the release build succeeds, verify that the correctly versioned Stack images have been pushed
 to Docker Hub.
 
+### Template Stack Release Process
+
+The Template Stacks we maintain are slightly different from the
+controller-based stacks that we maintain. Their processes are similar
+but a little simpler. This section will walk through how to release the
+Template Stacks themselves, and does not directly apply to core
+Crossplane.
+
+For Template Stacks, we do not use release branches unless we need to
+make a patch release. In the future we may need a more robust branching
+strategy, but for now we are not using branches because it is simpler.
+
+Note that Template Stacks **do not** require any code changes to update
+the version.
+
+### Template Stack Tag And Publish Pipeline
+
+Here is the list of all template stacks:
+
+* `stack-minimal-gcp`
+* `stack-minimal-aws`
+* `stack-minimal-azure`
+* `app-wordpress`
+
+Each one should be released as part of a complete release.
```

Comment:
I don’t think we need a change here, but despite us saying the releases of providers / stacks / apps, we still talk about them as though they are coupled. I don’t think this is necessarily a bad thing, but I would like to get to a point in the future where we just do new releases of non-core Crossplane repos either on their own cadences or just when needed 👍

Reply:
That is my understanding as well :)